src/containers/Layout/Layout.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Footer from '../../components/Layout/Footer/Footer'
+import Header from '../../components/Layout/Header/Header'
+import PropTypes from 'prop-types'
+
+export default class Layout extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      mobileMenu: false
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    this.setState({
+      mobileMenu: !this.state.mobileMenu
+    })
+  }
```

Comment:
Please refactor to initialize with an arrow function, as so:
<img width="669" alt="Screen Shot 2019-04-28 at 11 57 47 PM" src="https://user-images.githubusercontent.com/11988089/56878236-9c0d1680-6a42-11e9-9298-83576e6edd90.png">
Doing so will keep us from needing to bind `this` for the function, and we can use the state initializer syntax, without a constructor, which is more clean.